### PR TITLE
feat: log send to client error

### PR DESCRIPTION
**Pull request by maqi**

### CODE REVIEW

File: src/routing/mod.rs [hunk 1]
```diff
@@ -411,6 +411,7 @@ async fn handle_connection_events(
                 stage.send_event(event).await;
             }
             ConnectionEvent::Disconnected(addr) => {
+                info!("Lost connection to {:?}", addr);
```
Comment (madadam) [inline src/routing/mod.rs:414]: This is probably not important enough to use `info`, is it? I'd use `debug` or even `trace` to avoid spamming the logs.

**Review by madadam — APPROVED** [review]

File: src/routing/mod.rs (hunk 1 above)
Comment (maqi) [reply src/routing/mod.rs:414]: right, I will change it to trace

**Review by madadam — APPROVED** [review]

File: src/routing/mod.rs (hunk 1 above)
Comment (dirvine) [reply src/routing/mod.rs]: Nice catch @maqi  that kind of .is_ok() is possible and hides errors, so nice one.

File: src/routing/comm.rs [hunk 2]
```diff
@@ -105,7 +105,12 @@ impl Comm {
         msg: Bytes,
     ) -> Result<(), SendError> {
         if let Some(conn) = self.endpoint.get_connection(client) {
-            if conn.send_uni(msg.clone()).await.is_ok() {
+            if let Err(err) = conn.send_uni(msg.clone()).await {
```
Comment (dirvine) [inline src/routing/comm.rs:108]: We are still swallowing an error here, should we? I imagine as this returns a Result we need to return the error as a result here and not swallow it (prob why we have the issue in the first place)

File: src/routing/comm.rs [hunk 3]
```diff
@@ -105,7 +105,12 @@ impl Comm {
...
+                error!(
+                    "Sending message to client {:?} failed with error {:?}",
+                    client, err
+                );
```
Comment (dirvine) [inline src/routing/comm.rs:112]: possibly return Err(err) here

File: src/routing/comm.rs (hunk 2 above)
Comment (maqi) [reply src/routing/comm.rs:108]: This is to be addressed within the PR https://github.com/maidsafe/sn_routing/pull/2286

File: src/routing/comm.rs (hunk 3 above)
Comment (maqi) [reply src/routing/comm.rs:112]: This is to be addressed within the PR https://github.com/maidsafe/sn_routing/pull/2286

**Pull request merged**